src/pages/Categories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { statusCheck } from '../redux/categories/categories';
+
+const Categories = () => {
+  const status = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(statusCheck());
+        }}
+      >
```

Comment:
- [ ] Same here as well, it's better to create the function that handles the dispatch outside of the return and then call it here, kindly adjust that.